### DISCUSSION TITLE
TheLastBen's dreambooth colab no longer works for free users?

### DISCUSSION THREAD

**Discussion by bboytomato** [TheLastBen/fast-stable-diffusion#2268] [Q&A]
TheLastBen's dreambooth colab has been my go to for training faces using different existing models for free and even after the crackdown on webUIs on free colab users, I was glad it was still working properly. I even trained one just 2 weeks ago.
But today, I can no longer get it to work.
It just keeps showing an error message when running the training cell and so I did some reading around here.. I saw that it only runs with a VRAM of 20gb or higher now which is higher than the T4 gpu on colab, which is the free option (15gb). And I also noticed that on TheLastBen's github page, the colab link now has a (pro) label on it.
What happened? It used to work with just the free T4 gpu on colab.
Am I correct on this? Are there any alternatives (preferably free)?
Would really appreciate if anyone can help me out with this.
Thank you

**Comment by TheLastBen** [comment]
what is the error message ?

**Reply by bboytomato** [reply]
forgot to copy it earlier, will try to run again now and will get back to you with the error message asap. Thanks

**Reply by bboytomato** [reply]
Traceback (most recent call last):
  File "/content/diffusers/examples/dreambooth/train_dreambooth.py", line 803, in <module>
    main()
  File "/content/diffusers/examples/dreambooth/train_dreambooth.py", line 512, in main
    vae = AutoencoderKL.from_pretrained(args.pretrained_model_name_or_path, subfolder="vae")
  File "/usr/local/lib/python3.10/dist-packages/diffusers/models/modeling_utils.py", line 558, in from_pretrained
    raise ValueError(
ValueError: Cannot load <class 'diffusers.models.autoencoder_kl.AutoencoderKL'> from /content/stable-diffusion-custom because the following keys are missing: 
 decoder.mid_block.attentions.0.proj_attn.bias, encoder.mid_block.attentions.0.query.weight, decoder.mid_block.attentions.0.query.weight, encoder.mid_block.attentions.0.query.bias, encoder.mid_block.attentions.0.key.bias, decoder.mid_block.attentions.0.key.weight, decoder.mid_block.attentions.0.proj_attn.weight, encoder.mid_block.attentions.0.proj_attn.bias, decoder.mid_block.attentions.0.value.bias, encoder.mid_block.attentions.0.value.weight, encoder.mid_block.attentions.0.key.weight, decoder.mid_block.attentions.0.query.bias, encoder.mid_block.attentions.0.value.bias, decoder.mid_block.attentions.0.key.bias, decoder.mid_block.attentions.0.value.weight, encoder.mid_block.attentions.0.proj_attn.weight. 
 Please make sure to pass `low_cpu_mem_usage=False` and `device_map=None` if you want to randomely initialize those weights or else make sure your checkpoint file is correct.
Traceback (most recent call last):
  File "/usr/local/bin/accelerate", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.10/dist-packages/accelerate/commands/accelerate_cli.py", line 43, in main
    args.func(args)
  File "/usr/local/lib/python3.10/dist-packages/accelerate/commands/launch.py", line 837, in launch_command
    simple_launcher(args)
  File "/usr/local/lib/python3.10/dist-packages/accelerate/commands/launch.py", line 354, in simple_launcher
    raise subprocess.CalledProcessError(returncode=process.returncode, cmd=cmd)
subprocess.CalledProcessError: Command '['/usr/bin/python3', '/content/diffusers/examples/dreambooth/train_dreambooth.py', '--image_captions_filename', '--train_only_unet', '--save_starting_step=500', '--save_n_steps=0', '--Session_dir=/content/gdrive/MyDrive/Fast-Dreambooth/Sessions/rlgrdsddph', '--pretrained_model_name_or_path=/content/stable-diffusion-custom', '--instance_data_dir=/content/gdrive/MyDrive/Fast-Dreambooth/Sessions/rlgrdsddph/instance_images', '--output_dir=/content/models/rlgrdsddph', '--captions_dir=/content/gdrive/MyDrive/Fast-Dreambooth/Sessions/rlgrdsddph/captions', '--instance_prompt=', '--seed=398935', '--resolution=512', '--mixed_precision=fp16', '--train_batch_size=1', '--gradient_accumulation_steps=1', '--use_8bit_adam', '--learning_rate=2e-06', '--lr_scheduler=linear', '--lr_warmup_steps=0', '--max_train_steps=1350']' returned non-zero exit status 1.
Something went wrong

![image](https://github.com/TheLastBen/fast-stable-diffusion/assets/126328691/68aeffa4-1740-4f1d-a959-5ed9f0322cf8)


I did everything just as I've been doing ever since but for some reason, it always ends like this now.

**Reply by bboytomato** [reply]
also, here's screenshots of what I did.. don't know if this helps at all but i will share anyway just in case

![image](https://github.com/TheLastBen/fast-stable-diffusion/assets/126328691/d60d42b9-4333-4cf9-9140-070d706fb38e)

![image](https://github.com/TheLastBen/fast-stable-diffusion/assets/126328691/8ca2c8d9-e290-4641-b36f-faff6f21bfe4)
I already cropped the images by 512 but I checked the smartcrop box anyway since I ran into an error with the training cell before (around a month ago) and someone here posted a solution where checking it anyway somehow fixed it.

![image](https://github.com/TheLastBen/fast-stable-diffusion/assets/126328691/0ae301c7-6a02-4f0e-93a5-68a8178b4bd1)

I did everything exactly as I've been doing ever since but this time, it doesn't work.
Thanks in advance, TheLastBen

**Comment by TheLastBen** [comment]
what model are you using as base ?

**Reply by bboytomato** [reply]
dreamshaper shaper 4 w/ baked vae, fp16 (https://huggingface.co/Lykon/DreamShaper/blob/main/DreamShaper_4BakedVae_fp16.ckpt).
This is the same one I've been using for as long as I can remember. I'm really stumped on what's going wrong. I've also tried using a different computer to run the colab, same error.

**Reply by TheLastBen** [reply]
remove the link in path_to_huggingface and find a model with a non-baked vae

**Reply by bboytomato** [reply]
Hey man thanks for your response.. I actually tried using a link direct from civitai instead of doing the huggingface link and path (this is what i always did prior to running into this error) and it worked. Will give your solution a try as well and will let you know what happened. Thanks a lot!

**Reply by Jose082** [reply]
Hi, I faced the same issue. If you want to solve, just checkout to the DreamShaper b3a0cbb4471dad939676da196152d1012e81005b commit. Apparently the AutoencoderKL is not found in the latest version.

**Comment by arangodavid91** [comment]
Hi, I wanna know if it is still online for free users or I have to pay Stable Diffusion suscription. Thanks.

**Reply by TheLastBen** [reply]
dreambooth still works for free users

**Reply by gilroff** [reply]
@TheLastBen Is there any way for using automatic webui on free colab?

**Reply by TheLastBen** [reply]
google made sure that won't happen, sorry

**Reply by gilroff** [reply]
sad :(
